### PR TITLE
WT-4805 Modify test verification of error messages

### PR DESCRIPTION
**Pull request by kommiharibabu**
Modified the test to ensure that it verifies the returned
error messages for the operations

### CODE REVIEW

File: test/suite/test_prepare04.py [hunk 1]
```diff
@@ -96,8 +96,8 @@ def test_prepare_conflict(self):
         c.set_value(2)
         c.update()
         self.session.prepare_transaction('prepare_timestamp=' + self.prepare_ts)
-        conflictmsg = '/conflict between concurrent operations/'
-        preparemsg = '/conflict with a prepared update/'
+        conflictmsg = "WT_ROLLBACK: conflict between concurrent operations"
+        preparemsg = "WT_PREPARE_CONFLICT: conflict with a prepared update"
```
Comment (sueloverso) [inline test/suite/test_prepare04.py]: I think you don't need the change above. The form with `/string/` will match on that as a substring of the message so I don't think you need to make the message that much more detailed. It should be fine as-is.

File: test/suite/test_prepare04.py [hunk 2]
```diff
@@ -106,14 +106,17 @@ def test_prepare_conflict(self):
         s_other.begin_transaction(self.txn_config + self.ignore_config)
         c_other.set_key(1)
         if self.ignore == False and self.after_ts == True:
-            self.assertRaises(wiredtiger.WiredTigerError, lambda:c_other.search())
+            # WT_PREPARE_CONFLICT error returned when a read encounters a prepared transaction update
```
Comment (sueloverso) [inline test/suite/test_prepare04.py]: I would make this comment more general like:
`# Make sure we get the expected prepare conflict message.`

File: test/suite/test_prepare04.py [hunk 3]
```diff
@@ -106,14 +106,17 @@ def test_prepare_conflict(self):
...
+            self.assertRaisesException(wiredtiger.WiredTigerError, lambda:c_other.search(), preparemsg)
         else:
             c_other.search()
             self.assertTrue(c_other.get_value() == 1)
+
         c_other.set_value(3)
-        self.assertRaises(wiredtiger.WiredTigerError, lambda:c_other.update())
+
+        # WT_ROLLBACK error returned due to conflict with concurrent operations
+        self.assertRaisesException(wiredtiger.WiredTigerError, lambda:c_other.update(), conflictmsg)
```
Comment (sueloverso) [inline test/suite/test_prepare04.py]: Similar comment here.
`# Make sure we detect the conflict between operations.`

Comment (kommiharibabu) [reply test/suite/test_prepare04.py]: updated according to your comment.

File: test/suite/test_prepare04.py (hunk 2 above)
Comment (kommiharibabu) [reply test/suite/test_prepare04.py]: updated according to your comment.

File: test/suite/test_prepare04.py (hunk 1 above)
Comment (kommiharibabu) [reply test/suite/test_prepare04.py]: @sueloverso Thanks, changed the error strings as original substrings

**Pull request merged**